pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -666,6 +666,7 @@ def test_comparison_tzawareness_compat_scalars(self, op, box_with_array):
     # Raising in __eq__ will fallback to NumPy, which warns, fails,
     # then re-raises the original exception. So we just need to ignore.
     @pytest.mark.filterwarnings("ignore:elementwise comp:DeprecationWarning")
+    @pytest.mark.filterwarnings("ignore:Converting timezone-aware:FutureWarning")
```

Comment:
same question as on another PR, is this hiding something?

Reply:
AFAICT just noise in the logs